.changes/next-release/Feature-f0522fe2-1aa9-44fb-aa35-81aa9e78b52c.json

```
@@ -0,0 +1,4 @@
+{
+	"type": "Feature",
+	"description": "Code Transformation runs through Q Chat"
```

In any public materials, "Q" is always "Amazon Q".

```suggestion
	"description": "Code Transformation runs through Amazon Q Chat"
```